src/main/java/codechicken/nei/ItemHistoryPanel.java

```diff
@@ -39,8 +39,8 @@ public void update() {
 
     public void addItem(ItemStack stack) {
         if (stack != null) {
-            ItemStack is = StackInfo.loadFromNBT(StackInfo.itemStackToNBT(stack), 0);
-
+            ItemStack is = stack.copy();
```

Comment:
StackInfo is used here specifically to reset the fluid amount

remove this
![image](https://github.com/GTNewHorizons/NotEnoughItems/assets/31038811/09ab3ada-bda0-458e-9884-e2d2b1c3892e)